scripts/remove_dead_expressions.py

```
@@ -8,10 +8,12 @@
 BASE = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 def is_dead_seadex_ise(e):
+    """Check if expression is a disabled SeaDex ISE."""
     if not isinstance(e, dict): return False
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)